DittoToolsViewer/build.gradle.kts

```diff
@@ -34,7 +34,8 @@ dependencies {
 
     implementation(libs.live.ditto.ditto)
     implementation(libs.live.ditto.databrowser)
-    implementation(libs.live.ditto.exportlogs)
+//    implementation(libs.live.ditto.exportlogs)
+    implementation(project(":DittoExportLogs"))
```

Comment:
1. Don't forget to remove this before merging
2. This creates another issue that I think I've added to the tools backlog (to be figured out another time?), but if we uncomment out the code, it won't be pointing to the latest version of the tool - https://github.com/getditto/DittoAndroidTools/issues/123